distribution.pm

```diff
@@ -132,13 +132,14 @@ sub script_run {
         {
             timeout => $bmwqemu::default_timeout,
             output  => '',
-            quiet   => undef
+            quiet   => undef,
+            type_speed => testapi::get_var('VNC_TYPING_LIMIT')
```

Comment:
I would use `bmwqemu` directly similar to the line above, e.g. `$bmwqemu::vars{VNC_TYPING_LIMIT}`.

Reply:
Adapted it accordingly